dali/pipeline/executor/executor.cc

```diff
@@ -373,6 +379,7 @@ void Executor::PruneUnusedGraphNodes() {
 void Executor::SetupDataForGraph(WorkspaceBlob *wsb) {
   // Clear any old data setup
   wsb->Clear();
+  DeviceGuard g(device_id_);
```

Comment:
Maybe L382 -> L380, for consistency with L275 ?

Reply:
Fixed.